coffee/visitors/inspectors.py

```diff
@@ -554,6 +495,56 @@ def visit_Decl(self, o, ret=None, scope=LOCAL, *args, **kwargs):
         return ret
 
 
+class SymbolVisibility(Visitor):
+
+    @classmethod
+    def default_retval(cls):
+        return defaultdict(list), []
+
+    """
```

Comment:
Is this still recognised as a docstring, despite not being the first thing in the class?